tests/e2e/e2e_ibc_test.go

```diff
@@ -68,6 +70,39 @@ func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
 		// ibc hash of uatom token
 		uatomIBCHash := "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2"
 
+		// Get the uibc params for quota checking
+		uibcParams, err := s.umeeClient.QueryClient.QueryUIBCParams()
+		s.Require().NoError(err)
+		if uibcParams.IbcStatus != uibc.IBCTransferStatus_IBC_TRANSFER_STATUS_QUOTA_ENABLED {
```

Comment:
Why don't do something like this:
1. Start test with `s.Require.Equal(uibc.IBCTransferStatus_IBC_TRANSFER_STATUS_QUOTA_ENABLED, uibcParams.IbcStatus)`  -- this is the default, so should be set like that in the test
2. make a transfer in both (atom from gaia from umee, and uumee from umee to gaia)
3. make sure the balances got updated.
4. Make gov propo to change the IBC status to paused
5. make again transfers in both ways, and assure that neither balance got updated.

Reply:
Ok. I will change the tests flow